projects/landing-page/src/App.tsx

```diff
@@ -129,9 +129,16 @@ const App: React.FunctionComponent = () => {
                 Cumulus and HRMP
               </CardNetwork>
             </Grid>
-            <SectionRef href="https://github.com/paritytech/substrate-connect/tree/13bd26a1ca2904f8e0b5d04dfa35e82364d37d99/packages/connect/assets">
+            <SectionRef href="https://github.com/paritytech/substrate-connect/tree/main/packages/connect/src/connector/specs">
```

Comment:
Why do we have this link in the first place?
This should link to https://paritytech.github.io/substrate-connect/api/enums/connect_src.WellKnownChain.html for example, not to source code. Never link to source code from the documentation.